nan_json.h

```diff
@@ -78,14 +78,14 @@ class JSON {
     return scope.Escape(parse(json_string));
 #else
 #if NODE_MODULE_VERSION > NODE_6_0_MODULE_VERSION
```

Comment:
I thought for a moment that we should change this ^ to `#if NODE_MODULE_VERSION >= NODE_7_0_MODULE_VERSION`  ....     but I think it's OK as-is.